apps/condo/domains/organization/utils/accessSchema.js

```diff
@@ -1,7 +1,9 @@
+const { OrganizationLink } = require('./serverSchema')
 const { getByCondition } = require('@core/keystone/schema')
 
 async function checkOrganizationPermission (userId, organizationId, permission) {
     if (!userId || !organizationId) return false
+
     const employee = await getByCondition('OrganizationEmployee', {
```

Comment:
Please, use `OrganizationEmployee` from `serverSchema`.
Example usage in `apps/condo/domains/contact/access/Contact.js`

Reply:
done